docker.js

```diff
@@ -180,6 +179,32 @@ module.exports = {
             options.registry = app.config.driver.options?.registry || '' // use docker hub
         }
 
+        const networks = await this._docker.listNetworks({ filters: { label: ['com.docker.compose.network=flowforge'] } })
+        if (networks.length > 1) {
+            const filteredNetworks = []
+            for (let j = 0; j < networks.length; j++) {
+                const details = await this._docker.getNetwork(networks[j].Id).inspect()
+                const containers = Object.keys(details.Containers)
+                for (let i = 0; i < containers.length; i++) {
+                    // console.log(containers[i])
+                    if (containers[i].startsWith(process.env.HOSTNAME)) {
+                        filteredNetworks.push(networks[j])
+                    }
+                }
+            }
+            // console.log(JSON.stringify(filteredNetworks, null, 2))
+            if (filteredNetworks[0]) {
+                this._app.log.info(`[docker] using network ${filteredNetworks[0].Name}`)
+                this._network = filteredNetworks[0].Name
+            } else {
+                this._app.log.info('[docker] unable to find network')
+                process.exit(-9)
+            }
+        } else {
+            this._app.log.info(`[docker] using network ${networks[0].Name}`)
+            this._network = networks[0].Name
+        }
```

Comment:
```suggestion
        } else of (networks.length === 1) {
            this._app.log.info(`[docker] using network ${networks[0].Name}`)
            this._network = networks[0].Name
        } else {
            this._app.log.info('[docker] unable to find network')
            process.exit(-9)
        }
```

Ben i see there is a check + exit if non of the filtered networks match but there is no check for 0 networks. 

Is it worth adding a definitive test for `network.length` === 1 otherwise exit in the same manor?

Reply:
It shouldn't be possible if started with the docker-compose.yml, but might as well cover all options

Reply:
I though that might be the case and wanted to check with you.